src/app/project/project.component.ts

```diff
@@ -222,6 +222,11 @@ export class ProjectComponent implements OnInit {
         }
     }
 
+    // go to overview page
```

Comment:
Because this comment refers to the function its more common to do the "/** [...] */" comment

Reply:
true true, changed in [2832fce](https://github.com/dasch-swiss/dsp-app/pull/777/commits/2832fce9f9eb0e95528ce0c04e934986f136a34b)